web/src/routes/(user)/people/+page.svelte

```diff
@@ -194,7 +194,7 @@
 
   const handleMergePeople = async (detail: PersonResponseDto) => {
     await goto(
-      `${AppRoute.PEOPLE}/${detail.id}?${QueryParameter.ACTION}=${ActionQueryParameterValue.MERGE}&${QueryParameter.PREVIOUS_ROUTE}=${AppRoute.PEOPLE}`,
+      `${AppRoute.PEOPLE}/${detail.id}?${QueryParameter.ACTION}=${ActionQueryParameterValue.MERGE}&${QueryParameter.PREVIOUS_ROUTE}=${btoa(AppRoute.PEOPLE)}`,
```

Comment:
Why do we need `btoa` here now?

Reply:
Because, your position on the map is saved thanks to the url hash and you can't have `#` in a query parameter value. So I converted all `QueryParameter.PREVIOUS_ROUTE` values to base64 values

Reply:
You encode uri component for this